Header header

# This message defines the lost of control effectiveness of each actuator (0 .. 100%)

float64[6] LOE_true
float64[6] LOE_FDD
float64[6] LOE_calcul
